docs/topics/multiplatform-mobile/multiplatform-mobile-ios-dependencies.md

```diff
@@ -281,7 +283,8 @@ project depends on:
 * Multiplatform libraries that don't support the hierarchical structure.
 * Third-party iOS libraries, with the exception of [platform libraries](native-platform-libs.md) supported out of the
   box.
-  This issue applies only to the shared iOS source set. The IDE will correctly support the rest of the code.
+
+This issue applies only to the shared iOS source set. The IDE will correctly support the rest of the code.
```

Comment:
JFYI
`This issue ...` should be in next paragraph

Reply:
@danil-pavlov could you take a look?

Reply:
I agree, this sentence shouldn't be in the list